src/frontend/citizen-portal/src/app/components/disputant-form/disputant-form.component.html

```diff
@@ -0,0 +1,192 @@
+<form [formGroup]="form" novalidate>
+  <section class="mb-2">
+    <div class="row">
+      <div class="col-md-6">
+        <mat-form-field class="w-100">
+          <mat-label>Surname</mat-label>
+          <input matInput formControlName="disputant_surname" />
+          <mat-error>{{ "error.required" | translate }}</mat-error>
+        </mat-form-field>
+      </div>
```

Comment:
In this and the regular dispute stepper, please change the mat-lables for surname, given name to:
Legal Surname
Legal Given Name(s)

Reply:
done